src/index.html

```diff
@@ -10,13 +10,105 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo_link"
+        href="#"
+      >
+        <img
+          src="images/logo.jpg"
+          alt="Company Logo"
+          class="logo"
+        />
+      </a>
+      <nav>
```

Comment:
The `nav` tag should have a class name that represents its content according to the checklist.